packages/debug-utils/index.js

```diff
@@ -25,8 +25,8 @@ const commandReference = {
   "+": "add watch expression (`+:<expr>`)",
   "-": "remove watch expression (-:<expr>)",
   "?": "list existing watch expressions and breakpoints",
-  "b": "add breakpoint",
-  "B": "remove breakpoint",
+  "b": "add breakpoint (can specify line in several ways; see docs)",
```

Comment:
it'd be better if we provided some real hint vs. nothing. Maybe this is too much, I don't know:
```suggestion
  "b": "add breakpoint (`b [[<source-file>:]<line-number>]; see docs for more)",
```

Reply:
Oh that's nice; yeah, I'll change that.